roles/v2/pool/pool-config-example.toml

```diff
@@ -1,11 +1,23 @@
-listen_address = "0.0.0.0:34254"
-tp_address = "127.0.0.1:8442"
-listen_jn_address = "127.0.0.1:34264"
+# SRI Pool config
 authority_public_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"
 authority_secret_key = "2Z1FZug7mZNyM63ggkm37r4oKQ29khLjAvEx43rGkFN47RcJ2t"
 cert_validity_sec = 3600
 test_only_listen_adress_plain =  "0.0.0.0:34250"
+listen_address = "0.0.0.0:34254"
 # list of compressed or uncompressed pubkeys for coinbase payout (only supports 1 item in the array at this point)
 coinbase_outputs = [
     "04466d7fcae563e5cb09a0d1870bb580344804617879a14949cf22285f1bae3f276728176c3c6431f8eeda4538dc37c865e2784f3a9e77d044f33e407797e1278a",
 ]
+
+# SRI Pool JN config
+# local TP (must run a TP locally for this to run)
```

Comment:
The TP in this case doesn't relate to JN - the pool connects directly to the TP and opens up a listen port for the JN to connect to. So you can move the `SRI Pool JN config` comment down to the listen_jn_address as that is the only JN pool config field.